src/main/scala/sparkz/core/network/peer/PeerManager.scala

```diff
@@ -66,6 +66,12 @@ class PeerManager(settings: SparkzSettings, sparkzContext: SparkzContext) extend
         }
       peerDatabase.addOrUpdateKnownPeers(filteredPeers)
 
+    case AddToBlacklist(address) =>
+      peerDatabase.addToBlacklist(address, PenaltyType.MisbehaviorPenalty)
```

Comment:
do we wan't to add one more argument to `AddToBlacklist` where to specify the banning strategy?
For example, permanent or temporary (how much temporary).

Reply:
Yes, I think it's useful, I'll add it